core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainer.java

```diff
@@ -152,4 +173,195 @@ public String exportSystemErrors(boolean isPretty) {
     }
     return noticesByType;
   }
+
```

Comment:
The added code does not seem to be closely related to NoticeContainer - which is a container for notices, not an RTTI machine for schema generation. Please create a new class for the new code.